src/ui/modules/VideoHero.tsx

```
@@ -0,0 +1,311 @@
+          ) : (
+            <div className="absolute inset-0 flex items-center justify-center text-white">
+              <p>No thumbnail available</p>
+            </div>
+          )}
+          <div className="absolute inset-0 bg-black/30 flex items-center justify-center">
+            <button
+              className="w-16 h-16 bg-white rounded-full flex items-center justify-center"
+              aria-label="Play video"
+            >
+              {/* Play icon */}
+              <svg className="w-8 h-8" viewBox="0 0 24 24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)